docs/db/changelog/changesets/powerauth-java-server/1.5.x/20230723-add-table-unique-value.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <!--  Create table pa_unique_value -->
+    <changeSet id="1" logicalFilePath="powerauth-java-server/1.5.x/20230723-add-table-unique-value.xml" author="Roman Strobl">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="pa_unique_value"/>
+            </not>
+        </preConditions>
+        <comment>Create a new table pa_unique_value</comment>
+        <createTable tableName="pa_unique_value">
```

Comment:
Please describe this change in the migration guide as well.

Reply:
Good point, I will add it.